src/main/java/org/spongepowered/common/service/scheduler/AsyncScheduler.java

```diff
@@ -106,608 +51,139 @@
     private final Condition condition = this.lock.newCondition();
     // The dynamic thread pooling executor of asynchronous tasks.
     private ExecutorService executor;
-    // Query actor for task information
-    private SchedulerHelper schedulerHelper;
 
     private AsyncScheduler() {
```

Comment:
Does this need to be private?
Since it is async it does not really matter how much instances you create and use at the same time, does it? Sure Sponge itself only needs one but there is a difference between having one and enforcing a singleton.